### DISCUSSION TITLE
How to allow subdomain URLs in social icons

### DISCUSSION THREAD

**Discussion by goncalorazevedo** [hugo-sid/hugo-blog-awesome#76] [Q&A] [answered]
I am running into an odd issue and since I don't have a lot of knowledge on Hugo I've been having a hard time figuring it out.

Say I have this
```
[[params.socialIcons]]
name = "mastodon"
url = "mastodon.mydomain.com"

[[params.socialIcons]]
name = "github"
url = "github.mydomain.com"
```

In development it will show up as `127.0.0.1:1313/github.mydomain.com`.

Is there a workaround or is it not intended to be used like this and should there always be a protocol provided for it to consider absolute URLs?

Thank you so much in advance!

**Comment by hugo-sid** [comment, marked as answer]
Hi @vugonz. Protocol should be specified for `url` to considered as absolute URL.

Let me know if this works for you.
